cart/views.py

```diff
@@ -1,3 +1,92 @@
-from django.shortcuts import render
+import json
 
-# Create your views here.
+from django.views     import View
+from django.http      import JsonResponse
+
+from .models   import Cart
+
+from utils import login_decorator
+
+class CartListView(View):
+
+    @login_decorator
+    def post(self,request):
+        try:
+            data = json.loads(request.body)
+            user = request.user
+
+            if data['count'] <= 0:
+                return JsonResponse({"message": "LESS_THAN_MINIMUM_QUANTITY"}, status=400) 
+
+            if Cart.objects.filter(user=user, ingredient_id=data['ingredient_id']).exists():
+                cart = Cart.objects.get(user=user, ingredient_id=data['ingredient_id'])
+                cart.count += data['count']
+                cart.save()
+                return JsonResponse({'message':'ADDITIONAL_SUCCESS'}, status=200) 
+                    
+            Cart.objects.create(
+                count         = data['count'],
+                ingredient_id = data['ingredient_id'],
+                user          = user
+                )
```

Comment:
`get_or_create()` method를 사용하면 코드 수를 많이 줄일 수 있겠네요!
한번 검색해서 공부해보시고 적용해보시면 좋을 것 같아요! 만약 다른 기능도 구현해야되는게 남았다고 하면 2차에 사용해보세요~

```suggestion
    try:
            data          = data = json.loads(request.body)
            cart, created = get_or_create(user_id = request.user_id, ingredient_id = data['ingredient_id']) 
            cart.count += data["count"]                
            cart.save()                
            
            return JsonResponse({'message':'SUCCESS'}, status=200) 
            
    except:
            ..
```

Reply:
확인했습니다! 적용해보겠습니다!

Reply:
이번프로젝트 끝나고 추가로 공부하도록 하겠습니다. 감사합니다!!